src/utils/lint.ts

```diff
@@ -53,14 +53,17 @@ export async function lint(options?: Partial<DefaultOptions>) {
 
   let lintFiles: string[] = [];
   let prettierFiles: string[] = [];
+  let lintFilesPattern: string = '';
 
   if (mergedOptions.lintAll) {
     if (globalState.selectedSourceType === 'root') {
       lintFiles = glob.sync(`${globalState.projectRootPath}/{${srcPath.dir},${packagesPath.dir}}/**/*.{ts,tsx}`);
+      lintFilesPattern = `${globalState.projectRootPath}/{${srcPath.dir},${packagesPath.dir}}/**/*.{ts,tsx}`;
```

Comment:
Extract the common string `lintFiles` and `lintFilesPattern`  to a variable.

Reply:
done